ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingState.java

```diff
@@ -0,0 +1,6 @@
+package us.ihmc.behaviors.activeMapping;
+
+public enum ContinuousHikingState
```

Comment:
Different states, current NEED_TO_REPLAN, and PAUSED are not being used, wanted to leave them for now though